src/parser.ts

```diff
@@ -70,6 +78,22 @@ export function tokenize(str: string) {
       continue;
     }
 
+    // Special handling for commas
+    if (s === ',') {
+      // If we're in a number and not inside parenthesis include the comma
+      if (type === TokenType.NUM && parenDepth === 0) {
+        buffer += s;
```

Comment:
Would this work if we have numbers inside brackets? For example:
`10 * (a + 1,000)`